httpMiddleware.go

```diff
@@ -0,0 +1,199 @@
+package main
```

Comment:
The file should be using snake case as per Go conventions. So `htttp_middleware.go` instead of  `httpMiddleware.go`.

Reply:
I will do this when other comments are resolved

Reply:
https://github.com/lunarway/release-manager-bot/pull/21/commits/312071ea342b452a2dfd4c0bba649602f0a5a5b4